consensus/ibft/ibft.go

```diff
@@ -1026,6 +1026,13 @@ func (i *Ibft) insertBlock(block *types.Block) error {
 	// broadcast the new block
 	i.syncer.Broadcast(block)
 
+	// On every epoch start create checkpoint
+	if i.IsLastOfEpoch(block.Number()) {
```

Comment:
You can check this check inside hook

Reply:
Fixed ac388779230090c3ee8dc32f1fa312ecf14305e3